specification/index.bs

```diff
@@ -172,6 +175,12 @@ Issue(62): Specify localization handling.
 
 ## Match patterns
 
+A <dfn>match pattern</dfn> is a pattern used to match URLs.
```

Comment:
A pattern of what?

Reply:
Definitely agree we need to add more. This was just a stub that I hope to replace with more detail :)